requirements-base.txt

```diff
@@ -14,7 +14,7 @@ h5py==2.9.0
 numpy==1.17.0
 pandas==0.25.0
 pyBigWig==0.3.17
-pytest==4.4.1
+pytest>=5.4.3
```

Comment:
Received an internal Pytest ImpotError when running this version with the correct anaconda env for Python 3.6.